### PR TITLE
fix alembic migration for records_authors

### PR DESCRIPTION
**Pull request by MJedr**

### CODE REVIEW

File: backend/inspirehep/alembic/3637cb5551a8_update_values_in_authorschematype.py [hunk 1]
```diff
@@ -42,10 +42,44 @@
...
+        "ix_authors_records_record_id",
+        "tmp_records_authors",
+        ["record_id"],
+        unique=False,
+    )
+    op.create_index(
+        "ix_records_authors_id_type_authors_id",
+        "tmp_records_authors",
+        ["id_type", "author_id"],
+        unique=False,
```
Comment (drjova) [inline backend/inspirehep/alembic/3637cb5551a8_update_values_in_authorschematype.py:71]: shouldn't this be unique?

File: backend/inspirehep/alembic/3637cb5551a8_update_values_in_authorschematype.py [hunk 2]
```diff
@@ -42,10 +42,44 @@
...
+    )
+    op.create_index(
+        "ix_records_authors_id_type_record_id",
+        "tmp_records_authors",
+        ["id_type", "record_id"],
```
Comment (drjova) [inline backend/inspirehep/alembic/3637cb5551a8_update_values_in_authorschematype.py:76]: and this one?

File: backend/inspirehep/alembic/3637cb5551a8_update_values_in_authorschematype.py (hunk 1 above)
Comment (MJedr) [reply backend/inspirehep/alembic/3637cb5551a8_update_values_in_authorschematype.py:71]: this is copy-paste from the previous migrations, actually from my understanding it shouldn't be unique cause you can have same author with same kind of id (recid) in different records

File: backend/inspirehep/alembic/3637cb5551a8_update_values_in_authorschematype.py (hunk 2 above)
Comment (MJedr) [reply backend/inspirehep/alembic/3637cb5551a8_update_values_in_authorschematype.py:76]: again, it is copy-paste, and again I think it's correct cause you can have many records with same id_type but different value (so many authors, all have recids -> id_type)

**Pull request merged**